### PR TITLE
Ensure script returns error code from test execution

### PR DESCRIPTION
**Pull request by bparees**
## Description
the trailing echo command in the script lead to it exiting with code 0 instead
of the error code from the make test-e2e invocation, which in turn lead to
prow always considering the job to have succeeded.

## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: tests/e2e/test_api.py [hunk 1]
```diff
@@ -42,7 +42,8 @@ def test_invalid_question() -> None:
         "/v1/query", json={"conversation_id": "1234", "query": "test query"}, timeout=20
     )
     print(vars(response))
-    assert response.status_code == requests.codes.unprocessable
+    #assert response.status_code == requests.codes.unprocessable
+    assert response.status_code == 100
```
Comment (bparees) [inline tests/e2e/test_api.py:46]: deliberate error for testing the job, will remove before merge

File: scripts/test-e2e-standalone.sh [hunk 2]
```diff
@@ -58,4 +58,3 @@ fi
 
 echo Done waiting for OLS server start, running e2e
 make test-e2e
```
Comment (tisnik) [inline scripts/test-e2e-standalone.sh:60]: `set -e` at beginning of script might do the magic

Comment (bparees) [reply scripts/test-e2e-standalone.sh:60]: yeah i considered it and may still but i've been burned before by the surprising effects of set -e when you don't want them. For now this should be sufficient i think.

**Pull request closed without merging**